Makefile.defs

```diff
@@ -449,7 +449,7 @@ else
 		CSTD_FLAGS:=-std=c11
 		CXXSTD_FLAGS:=-std=c++11
 	else
-		# Keep GNU extensions enabled on Linux (mainly because the CI's Clang version sucks so very hard).
+		# Keep GNU extensions enabled on Linux, otherwise Clang < 9 fails to build LuaJIT.
```

Comment:
But how do we know that's an accurate comment? :-P

Reply:
https://circleci.com/gh/NiLuJe/koreader-base/1054?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link ;p

Reply:
Rather, I mean who says Clang 9 does?

Reply:
I do ;p.

```
clang  -O2 -fomit-frame-pointer -Wall  -std=c11 -march=native -pipe -O2 -fomit-frame-pointer -frename-registers -fweb -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -U_FORTIFY_SOURCE  -DLUA_MULTILIB=\"lib\" -fno-stack-protector   -c -o ljamalg.o ljamalg.c
clang-9: warning: optimization flag '-frename-registers' is not supported [-Wignored-optimization-argument]
clang-9: warning: optimization flag '-fweb' is not supported [-Wignored-optimization-argument]
gcc-ar rcus 2>/dev/null libluajit.a lj_vm.o ljamalg.o
clang  -O2 -fomit-frame-pointer -Wall  -std=c11 -march=native -pipe -O2 -fomit-frame-pointer -frename-registers -fweb -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -U_FORTIFY_SOURCE  -DLUA_MULTILIB=\"lib\" -fno-stack-protector    -Wl,-O1 -Wl,--sort-common -Wl,--hash-style=gnu -Wl,--as-needed -shared -fPIC -Wl,-soname,libluajit-5.1.so.2   -o libluajit.so lj_vm_dyn.o ljamalg_dyn.o -lm -ldl  
gcc-ranlib libluajit.a
strip --strip-unneeded libluajit.so
clang  -O2 -fomit-frame-pointer -Wall  -std=c11 -march=native -pipe -O2 -fomit-frame-pointer -frename-registers -fweb -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -U_FORTIFY_SOURCE  -DLUA_MULTILIB=\"lib\" -fno-stack-protector    -Wl,-O1 -Wl,--sort-common -Wl,--hash-style=gnu -Wl,--as-needed  -Wl,-E   -o luajit luajit.o libluajit.a -lm -ldl  
strip --strip-unneeded luajit
make[2]: Leaving directory '/var/tmp/niluje/luajit-2.0/src'
```